src/NUnitFramework/framework/Has.cs

```diff
@@ -99,7 +99,21 @@ public static ItemsConstraintExpression Exactly(int expectedCount)
         {
             return new ConstraintExpression().Exactly(expectedCount);
         }
- 
+
+        #endregion
+
+        #region One
+
+        /// <summary>
+        /// Returns a <see cref="ItemsConstraintExpression"/> which will apply
+        /// the following constraint to only one member of the collection,
+        /// and fail if none or more than one match occurs.
+        /// </summary>
+        public static ItemsConstraintExpression One()
```

Comment:
Once again, the issue specifies `Has.One` not `Has.One()`. Is there any reason this can't be a property?